task_3/theater-no-orm/pkg/data/theater.go

```diff
@@ -0,0 +1,427 @@
+package data
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type Account struct {
+	Id          int
+	FirstName   string
+	LastName    string
+	PhoneNumber string
+	Email       string
+}
+
+type Ticket struct {
```

Comment:
could you please create separate `.go` file for each entity?
So, instead of just `theater.go` should be: `account.go`, `ticket.go` and etc..